bin/shpec

```diff
@@ -36,11 +36,30 @@ end_describe() {
 # any identifier as a function name.
 
 stub_command() {
+  local func_decl="$(get_function_declaration "$1")"
+  [ -n "${func_decl}" ] && eval "_shpec____${func_decl}"
```

Comment:
I'm not referring to `func_decl`, I'm referring to the function that is `eval`d here. After calling `stub_command foo; unstub_command foo` there will be still be a `_shpec___foo` function in the namespace, correct?